hubblestack_pulsar/hubblestack_pulsar_win_config.yaml

```diff
@@ -8,6 +8,19 @@ C:\Windows:
     - TakeOwnership
   exclude:
     - C:\Windows\System32
+    - C:\Windows\security\logs\diagnosis.log
+    - C:\Windows\ServiceProfiles\LocalService\AppData\Local\lastalive?.dat
+    - C:\Windows\WinSxS\FileMaps
+    - C:\Windows\security\logs
+    - C:\Windows\ServiceProfiles\LocalService\AppData\Local
+    - C:\Windows\security\templates\policies
+    - C:\Windows\CCM\Logs
+    - C:\Users\*\AppData\Local\VMware\vpx
+    - C:\Windows\security\database
+    - C:\Users\*\AppData\Local\Mozilla\Firefox\Profiles
+    - C:\Users\*\AppData\Local\Temp
+    - C:\Windows\SoftwareDistribution
+    - C:\Windows\WinSxS
 C:\temp: {}
```

Comment:
Pretty sure it was just on the list of "hey, here are some windows paths". I don't remember who created the original file. But I'm with @anuragpaliwal80, we probably don't need to watch temp